docs/es/README.md

```diff
@@ -162,3 +173,7 @@ Obtenga más información en los siguientes enlaces, que han sido aportados por
 ## Mantenedores
 
 - [Felix Angelov](https://github.com/felangel)
+
+## Partidarios
+
+[<img src="https://raw.githubusercontent.com/felangel/bloc/master/docs/assets/vgv_logo.png" width="120" />](https://verygood.ventures)
```

Comment:
super minor but I've updated the README to move the "Sponsors" section higher up and to include stream. Can we adjust this to be consistent with https://github.com/felangel/bloc/blob/master/README.md? Thanks! 🙏

Reply:
Sure, no problem!